feature-flag/src/test/kotlin/com/linecorp/android/featureflag/loader/FeatureFlagOptionParserTest.kt

```diff
@@ -50,7 +51,10 @@ object FeatureFlagOptionParserTest : Spek({
         context("with normal case") {
             val expectedResults = listOf(
                 setOf(PRIVATE),
-                setOf(PRIVATE, OVERRIDABLE)
+                setOf(PRIVATE, OVERRIDABLE),
+                setOf(DEPRECATED),
+                setOf(PRIVATE, DEPRECATED),
+                setOf(PRIVATE, OVERRIDABLE, DEPRECATED)
```

Comment:
Please add cases for "with duplicated option" as well.

Reply:
Done